client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditTable.tsx

```diff
@@ -54,18 +56,20 @@ const PlaceholderRow = ({
   return quantity === null ? null : (
     <tr>
       <PlaceholderCell
-        colSpan={4 + extraColumnOffset}
+        colSpan={5 + extraColumnOffset}
         sx={{ color: 'secondary.main' }}
       >
         {t('label.placeholder')}
       </PlaceholderCell>
-      <PlaceholderCell
-        style={{ textAlign: 'right', paddingRight: '14px' }}
-        colSpan={2}
-      >
+      <PlaceholderCell style={{ textAlign: 'right', paddingRight: '14px' }}>
         1
       </PlaceholderCell>
-      <PlaceholderCell colSpan={3}></PlaceholderCell>
+      {!!dosesPerUnit && (
+        <PlaceholderCell style={{ textAlign: 'right', paddingRight: '14px' }}>
+          {dosesPerUnit}
+        </PlaceholderCell>
```

Comment:
thought it was helpful to show the default doses per unit in the placeholder, to help see the relationship between that and units